packages/@guidesmiths/cuckoojs-schematics/package.json

```diff
@@ -42,7 +42,7 @@
   "schematics": "./dist/collection.json",
   "scripts": {
     "build": "tsc -p tsconfig.json",
-    "postbuild": "../../../node_modules/.bin/copyfiles -a -u 1 ./src/collection.json ./src/*/schema.json ./src/*/files/** ./src/*/files/*/** dist/",
+    "postbuild": "../../../node_modules/.bin/copyfiles -a -u 1 ./src/collection.json ./src/*/schema.json ./src/*/files/** ./src/*/files/*/** ./src/*/files/*/*/** dist/",
```

Comment:
If we add more schematics that don't follow any of these patterns, we will need to keep adding more to this script. Would it make sense to have a folder for all the `files` that need to be copied?

Reply:
I did a quick search based on that comment and I found this: https://ls-lint.org/ Could it be interesting?¿

Reply:
Mmmmm. I didn't know that library and I'm really fan of linters. So it could be a nice improvement :)

Reply:
I've added a ticket to investigate this further